### PR TITLE
refactor(examples) Update `quickstart-pytorch` example 

### PR DESCRIPTION
**Pull request by jafermarq**
Update `examples/quickstart-pytorch` to work with `flwr run`.

### CODE REVIEW

File: examples/quickstart-pytorch/README.md [hunk 1]
```diff
@@ -8,90 +8,57 @@ framework: [torch, torchvision]
...
-```
-
-If you don't see any errors you're good to go!
-
-______________________________________________________________________
-
-## Run Federated Learning with PyTorch and Flower
-
-Afterwards you are ready to start the Flower server as well as the clients. You can simply start the server in a terminal as follows:
+This will create a new directory called `quickstart-mlx` with the following structure:
```
Comment (jafermarq) [inline examples/quickstart-pytorch/README.md:24]:
```suggestion
This will create a new directory called `quickstart-pytorch` with the following structure:
```

File: examples/quickstart-pytorch/pytorchexample/client_app.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        return loss, len(self.valloader.dataset), {"accuracy": accuracy}
+
+
+def client_fn(context: Context):
+    """Construct a Client that will be run in a ClientApp."""
+
+    # Read the node_config to fetch data partition associated to this node
+    partition_id = context.node_config["partition-id"]
+    num_partitions = context.node_config["num-partitions"]
+    # Read run_config to fetch hyperparameters relevant to this run
```
Comment (danieljanes) [inline examples/quickstart-pytorch/pytorchexample/client_app.py:53]:
```suggestion
    num_partitions = context.node_config["num-partitions"]

    # Read run_config to fetch hyperparameters relevant to this run
```

**Review by danieljanes — APPROVED** [review]
